app/controllers/admin/resources_controller.rb

```diff
@@ -39,7 +39,39 @@ def destroy
       else
         redirect_to admin_conference_resources_path(conference_id: @conference.short_title),
                     error: 'Resource was successfully destroyed.' \
-  	                "#{@resource.errors.full_messages.join('. ')}."
+                    "#{@resource.errors.full_messages.join('. ')}."
+      end
+    end
+
+    def increment_quantity
+      @resource.quantity = @resource.quantity + 1
+      @resource.save
+      respond_to do |format|
+        format.js{ render action: 'update_quantity' }
+      end
+    end
+
+    def decrement_quantity
```

Comment:
`increment_quantity`, `decrement_quantity` are exactly the same method except by the fact that one increment the variable and the other one decrements it. to avoid repeating code what about having only `increment_quantity` with a parameter, something like:


``` Ruby
  def increment_quantity(step = 1)
      @resource.quantity = @resource.quantity + step
      @resource.save
      respond_to do |format|
        format.js{ render action: 'update_quantity' }
      end
    end
```

and then

`increment_quantity` and `increment_quantity(-1)`

Moreover this would allow to change the increment step easily in case that is needed.